CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
 
-set(IRODS_MINIMUM_VERSION "4.90.0")
+set(IRODS_MINIMUM_VERSION "5.0.0")
 find_package(IRODS "${IRODS_MINIMUM_VERSION}" REQUIRED)
 set(IRODS_PLUGIN_REVISION "0")
 set(IRODS_PLUGIN_VERSION "${IRODS_MINIMUM_VERSION}.${IRODS_PLUGIN_REVISION}")
```

Comment:
I'm only going to ask this question on this PR, but the same applies for all the plugins...

Are we going to keep the versioning scheme the same for 5? The version number which will result for the plugins (or, at least this one) will be "5.0.0.0".

As I understand it, we are aiming to make the plugins compatible with any given minor version of the server, and I thought we were also trying to decouple the plugin versioning from the server versioning (a la pam_interactive).

Reply:
For the first 5.0 release, the approach is to bump all versions to 5.0 like we'd normally do. That would at as our soft-reset on the version numbers. Then, all releases following 5.0 would move forward with decoupled versioning.

That's how I envisioned things going.

Reply:
Gotcha. How do we feel about dropping the plugin revision, then? New version is 5.0.0? Can we even do that?

Reply:
I don't see see an issue with doing that.

Will add that to the discush list.

Reply:
After some discussion, we've decided to keep the revision number for plugins.